cucumber/api/features/support/rest_helpers.rb

```diff
@@ -173,13 +187,27 @@ def create_user login, owner
       @user_index += 1
     end
 
-    return if users[login]
-
     roleid = "cucumber:user:#{login}"
-    Role.create(role_id: roleid).tap do |user|
-      Credentials[role: user] || Credentials.new(role: user).save(raise_on_save_failure: true)
+    create_role(roleid, owner)
+  end
+
+  # Create a regular host, owned by the admin user
+  def create_host login, owner
+    unless login
```

Comment:
doesn't look like login will ever not be supplied so let's look into why we need the `unless login`

Reply:
thanks, removed this logic.